code/datums/mutations.dm

```diff
@@ -122,18 +123,37 @@ var/thanks_tobba = 'icons/fonts/runescape_uf.ttf'
 	..()
 	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_f_s", "layer"=-MUTATIONS_LAYER)
 	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_m_s", "layer"=-MUTATIONS_LAYER)
+	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_moth_f_s", "layer"=-MUTATIONS_LAYER)
+	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_moth_m_s", "layer"=-MUTATIONS_LAYER)
+	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_skeleton_s", "layer"=-MUTATIONS_LAYER) // god why doesn't this work
+	visual_indicators |= image("icon"='icons/effects/genetics.dmi', "icon_state"="hulk_alien_s", "layer"=-MUTATIONS_LAYER)
 
 /datum/mutation/human/hulk/on_acquiring(mob/living/carbon/human/owner)
 	if(..())
 		return
 	var/status = CANSTUN | CANWEAKEN | CANPARALYSE | CANPUSH
 	owner.status_flags &= ~status
+	if(MUTCOLORS in owner.dna.species.specflags)
+		naturalcolor = owner.client.prefs.features["mcolor"]
+		var/new_mutantcolor = "#3DCF13" as color
+		owner.client.prefs.features["mcolor"] = sanitize_hexcolor(new_mutantcolor)
+		owner.regenerate_icons()
 
 /datum/mutation/human/hulk/on_attack_hand(mob/living/carbon/human/owner, atom/target)
 	return target.attack_hulk(owner)
 
 /datum/mutation/human/hulk/get_visual_indicator(mob/living/carbon/human/owner)
-	var/g = (owner.gender == FEMALE) ? 1 : 2
+	var/g = 1
+	var/spec = owner.dna.species.id
+	if(spec == "human")
+		g = (owner.gender == FEMALE) ? 1 : 2
+	else if(spec == "moth")
+		g = (owner.gender == FEMALE) ? 3 : 4
+	//else if(spec == "skeleton")
+	//	g = 5 // this doesn't work and i have no idea why so i'm commenting it out until i can make it work.
```

Comment:
it just doesn't actually do anything, they don't look any different and I dunno why.